### PR TITLE
{bio}[GCC/9.3.0] freebayes v1.3.5 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)
Requires vcflib (#12841) and SeqLib (#12856)

### CODE REVIEW

File: easybuild/easyconfigs/f/freebayes/freebayes-1.3.5_find-external-libs.patch [hunk 1]
```diff
@@ -0,0 +1,49 @@
+diff -Nru freebayes-1.3.5.orig/meson.build freebayes-1.3.5/meson.build
```
Comment (Micket) [inline easybuild/easyconfigs/f/freebayes/freebayes-1.3.5_find-external-libs.patch:1]: Could you please add a short description + a note of who the author is (probably you?) in all the patches?

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/freebayes/freebayes-1.3.5_find-external-libs.patch (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/f/freebayes/freebayes-1.3.5_find-external-libs.patch:4]: done

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 12 out of 14 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.4
See https://gist.github.com/9715fed5dbe305d3db18fe67519794d6 for a full test report.

**Comment by jfgrimm** [comment]
@verdurin you need to rebuild BWA, since the easyblock was updated

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.4
See https://gist.github.com/b62c2a21ec717f9b98f91b2b58631225 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 15 out of 15 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/4cd89bdcde203c8a577950cfc710c478 for a full test report.

**Comment by verdurin** [comment]
@Micket are you happy with the changes now?

**Review by Micket — APPROVED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/47b42dcaa785ab8ebcab1afd7817c300 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**